hazelcast/src/main/java/com/hazelcast/internal/bootstrap/TpcServerBootstrap.java

```diff
@@ -120,39 +124,84 @@ public void start() {
     }
 
     private void startNio() {
+        AltoSocketConfig clientSocketConfig = getClientSocketConfig();
+
+        if (clientSocketConfig == null) {
```

Comment:
I think we should throw an error for now because effectively no client can connect to alto even though the only purpose (for now) is that only clients can connect to it.

Reply:
https://github.com/hazelcast/hazelcast/pull/23254/commits/9b62315c9c65348eb05984f7b2aa21538689f324